catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -111,7 +111,7 @@ define([
                     return;
                 }
             } else {
-                if (this.isEmpty() && !this.get('regex') && !this.get('required')) {
+                if (!this.get('value') && !this.get('regex') && !this.get('required')) {
```

Comment:
Sounds good.  I'm wondering if it might be worth it to split this into a separate `if` specifically for type 'point' so it doesn't affect the other types.